app/models/spree/payment/processing.rb

```diff
@@ -193,6 +199,20 @@ def gateway_options
 
       private
 
+      def ready_to_process?
```

Comment:
:heart: the comment-method pattern.

Reply:
😁 Maikel gets credit for pointing out the original flag-param anti-pattern.